src/PrestaShopBundle/Form/Admin/Sell/Product/Combination/BulkCombinationStockType.php

```diff
@@ -156,6 +157,23 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
                 'disabling_switch' => true,
             ])
+            ->add('available_now_label', TranslatableType::class, [
```

Comment:
Add constraints (both in now and later label).  CombinationSettings class doesn't exist yet, refer to comment above for more info https://github.com/PrestaShop/PrestaShop/pull/29411/files#r956046238 . smth like:
```suggestion
            ->add('available_now_label', TranslatableType::class, [
                'options' => [
                    constraints' => [
                        new Length([
                            'max' => CombinationSettings::AVAILABLE_NOW_MAX_LENGTH
                        ]),
                        new TypedRegex(TypedRegex::TYPE_GENERIC_NAME),
                    ]
                ]
```

Reply:
not sure if im not forgetting something and if bulkCombination form handles form constraint errors (but i think it should :thinking: )

Reply:
done